packages/appium/lib/cli/parser.js

```diff
@@ -258,19 +253,16 @@ class ArgParser {
 }
 
 /**
- * Creates a {@link ArgParser} instance.  Necessarily reads extension configuration
- * beforehand, and finalizes the config schema.
+ * Creates a {@link ArgParser} instance; finalizes the config schema.
  *
  * @constructs ArgParser
  * @param {boolean} [debug] - If `true`, throw instead of exit upon parsing error
- * @returns {Promise<ArgParser>}
+ * @returns {ArgParser}
  */
-async function getParser (debug = false) {
-  await B.all([driverConfig.read(), pluginConfig.read()]);
+function getParser (debug) {
```

Comment:
why the default value has been removed?

Reply:
it was redundant; `getParser()` is the only code which calls `new ArgParser()` and the signature for the `ArgParser` constructor is `constructor(debug = false)`.  So the default gets set there.  This prevents us from needing to maintain two sets of the same default value.